### PR TITLE
delete realization 36_buildings/services_putty and 36_buildings/services_with_capital

### PR DESCRIPTION
**Pull request by LaviniaBaumstark**
## Purpose of this PR
cleaning up REMiND: 
delete old realizations 36_buildings/services_putty and 36_buildings/services_with_capital and all releated code in 29_CESparameters/calibrate

## Type of change

- [x] Refactoring

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [x] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [x] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: /p/tmp/lavinia/MO/test_cleaningRealizations/remind/output, including calibration
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 1]
```diff
@@ -371,193 +337,27 @@ loop (ttot$( ttot.val ge 2005),
...
-  loop((t_29scen(t),regi_dyn29(regi),in)$(   ppfen_industry_dyn37(in)
-                                          OR ppfKap_industry_dyn37(in) ),
-    put "%c_expname%", "target", t.tl, regi.tl, "quantity", in.tl;
-    put pm_cesdata(t,regi,in,"quantity") /;
-  );
-$endif.industry_FE_target
-$endif.subsectors
-
-  putclose file_CES_calibration;
-);
```
[the remark this answers is not in the document]
Comment (LaviniaBaumstark) [reply modules/29_CES_parameters/calibrate/preloop.gms:521]: I will add it

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — CHANGES_REQUESTED** [review]

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 2]
```diff
@@ -755,33 +486,33 @@ if (smax((t,regi_dyn29(regi)),
     sm_tmp   !! by how much does en + kap exceed inco?
     = ( ( pm_cesdata(t,regi,"en","quantity")
         * pm_cesdata(t,regi,"en","price")
-	)
+    )
       + ( pm_cesdata(t,regi,"kap","quantity")
         * pm_cesdata(t,regi,"kap","price")
-	)
+    )
       )
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:493]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 3]
```diff
@@ -755,33 +486,33 @@ if (smax((t,regi_dyn29(regi)),
...
         )
       / ( ( pm_cesdata(t,regi,"en","quantity")
           * pm_cesdata(t,regi,"en","price")
-	  )
+      )
         + ( pm_cesdata(t,regi,"kap","quantity")
           * pm_cesdata(t,regi,"kap","price")
-	  )
+      )
         );
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:516]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 4]
```diff
@@ -816,20 +547,20 @@ loop (cesOut2cesIn("inco",in)$( NOT sameas(in,"lab") ),
 
       if (sm_tmp gt 1,
         put "  ", t.tl, " ", regi.tl, in.tl:>4, "   ",
-	    pm_cesdata(t,regi,in,"quantity"), " x ",
-	    pm_cesdata(t,regi,in,"price"), " > ",
-	    pm_cesdata(t,regi,"inco","quantity"), " -> ";
+         pm_cesdata(t,regi,in,"quantity"), " x ",
+         pm_cesdata(t,regi,in,"price"), " > ",
+         pm_cesdata(t,regi,"inco","quantity"), " -> ";
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:552]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 5]
```diff
@@ -816,20 +547,20 @@ loop (cesOut2cesIn("inco",in)$( NOT sameas(in,"lab") ),
...
-	put pm_cesdata(t,regi,in,"price") /;
+    = ( pm_cesdata(t,regi,"inco","quantity")
+      - sum(cesOut2cesIn2("inco",in2)$( NOT sameas(in,in2) ),
+          pm_cesdata(t,regi,in2,"quantity")
+        * pm_cesdata(t,regi,in2,"price")
+        )
+      )
+    / pm_cesdata(t,regi,in,"quantity");
+
+    put pm_cesdata(t,regi,in,"price") /;
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:563]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 6]
```diff
@@ -926,17 +657,15 @@ if (card(ppf_beyondcalib_29) >= 1, !! if there are any nodes in beyond calib
       * p29_cesdata_load(t,regi,in,"eff")
       * p29_effGr(t,regi,in)
 
-      * ( p29_cesIO_load(t,regi,out)$( NOT ipf_putty(out) )
-        + p29_cesIOdelta_load(t,regi,out)$( ipf_putty(out) )
+      * ( p29_cesIO_load(t,regi,out)
         )
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:661]: What's with the brackets?

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 7]
```diff
@@ -926,17 +657,15 @@ if (card(ppf_beyondcalib_29) >= 1, !! if there are any nodes in beyond calib
...
       ** (1 - p29_cesdata_load(t,regi,out,"rho"))
 
       * exp(
         log(
         p29_cesdata_load(t,regi,in,"eff")
         * p29_effGr(t,regi,in)
-        * ( p29_cesIO_load(t,regi,in)$( NOT ipf_putty(out))
-          + p29_cesIOdelta_load(t,regi,in)$( ipf_putty(out))
+        * ( p29_cesIO_load(t,regi,in)
           )
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:669]: What's with the brackets?

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 8]
```diff
@@ -1056,17 +777,11 @@ $endif.subsectors
...
-        if ( ipf_putty(out),
-          put pm_cesdata_putty.tn(t,regi,in,"quantity"), " = ";
-          put pm_cesdata_putty(t,regi,in,"quantity") /;
-        );
 
-        put pm_cesdata.tn(t,regi,in,"price"), " = ";
-        put pm_cesdata(t,regi,in,"price") /;
+           put pm_cesdata.tn(t,regi,in,"price"), " = ";
+           put pm_cesdata(t,regi,in,"price") /;
       );
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:785]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 9]
```diff
@@ -1336,13 +919,13 @@ if (0 gt sm_tmp,
          cesOut2cesIn(ue_industry_dyn37(out),ppfKap(in))),
     sm_tmp = pm_cesdata(t,regi,out,"quantity")
            - ( pm_cesdata(t,regi,in,"quantity")
-	     * pm_cesdata(t,regi,in,"price")
-	     );
+         * pm_cesdata(t,regi,in,"price")
+         );
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:923]: Fix indentation.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 10]
```diff
@@ -1541,13 +1106,9 @@ loop ((out,in,in2,t)$((pm_cesdata_sigma(t,out) eq -1)
...
-       );
-
-       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, out.tl   , "quantity", regi.tl, ( pm_cesdata(t,regi,out,"quantity") $ ( NOT ipf_putty(out)) + pm_cesdata_putty(t,regi,out,"quantity") $ ( ipf_putty(out))) /;
-       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, in.tl , "quantity", regi.tl, ( pm_cesdata(t,regi,in,"quantity") $ ( NOT ipf_putty(out)) + pm_cesdata_putty(t,regi,in,"quantity") $ ( ipf_putty(out))) /;
-       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, in2.tl  , "quantity", regi.tl, ( pm_cesdata(t,regi,in2,"quantity") $ ( NOT ipf_putty(out)) + pm_cesdata_putty(t,regi,in2,"quantity") $ ( ipf_putty(out))) /;
+       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, out.tl   , "quantity", regi.tl, ( pm_cesdata(t,regi,out,"quantity") ) /;
+       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, in.tl , "quantity", regi.tl, ( pm_cesdata(t,regi,in,"quantity") ) /;
+       put sm_CES_calibration_iteration:0:0, "remind" , t.tl, in2.tl  , "quantity", regi.tl, ( pm_cesdata(t,regi,in2,"quantity") ) /;
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline modules/29_CES_parameters/calibrate/preloop.gms:1111]: What's with the brackets?

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — CHANGES_REQUESTED** [review]

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — CHANGES_REQUESTED** [review]
https://github.com/remindmodel/remind/pull/1509/files#diff-54a5960804a16adc77b7ddc96e32be89e442abbd1f89b0fa5cd8e7c5a1b174b7R714

Probably some years-old merge leftover, but that comment should go, too.

File: modules/29_CES_parameters/calibrate/preloop.gms (hunk 7 above)
Comment (LaviniaBaumstark) [reply modules/29_CES_parameters/calibrate/preloop.gms:669]: closing the log

**Comment by LaviniaBaumstark** [comment]
> https://github.com/remindmodel/remind/pull/1509/files#diff-54a5960804a16adc77b7ddc96e32be89e442abbd1f89b0fa5cd8e7c5a1b174b7R714
> 
> Probably some years-old merge leftover, but that comment should go, too.

I added teh comment, what do you mean?

File: modules/29_CES_parameters/calibrate/preloop.gms (hunk 2 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply modules/29_CES_parameters/calibrate/preloop.gms]: Close enough.

File: modules/29_CES_parameters/calibrate/preloop.gms [hunk 11]
```diff
@@ -1323,11 +906,11 @@ $ifthen.industry_FE_target "%c_CES_calibration_industry_FE_target%" == "1"
...
-	   pm_cesdata(t,regi,out,"quantity")
-	 - ( pm_cesdata(t,regi,in,"quantity")
-	   * pm_cesdata(t,regi,in,"price")
-	   )
-	);
+       pm_cesdata(t,regi,out,"quantity")
+     - ( pm_cesdata(t,regi,in,"quantity")
+       * pm_cesdata(t,regi,in,"price")
+       )
+    );
```
[the remark this answers is not in the document]
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply modules/29_CES_parameters/calibrate/preloop.gms]: Is it so difficult to properly indent code?

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
https://github.com/remindmodel/remind/blob/5c197bd42b7f4fc66f1bc9073896f632ea656f8a/modules/29_CES_parameters/calibrate/preloop.gms#L992

File: modules/29_CES_parameters/calibrate/preloop.gms (hunk 5 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply modules/29_CES_parameters/calibrate/preloop.gms]: > * Use indentations to make the structure more readable

[[↑]](https://github.com/remindmodel/remind/blob/5c197bd42b7f4fc66f1bc9073896f632ea656f8a/main.gms#L144)

Why is the assignment not indented with the thing being assigned to?

File: modules/29_CES_parameters/calibrate/preloop.gms (hunk 6 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply modules/29_CES_parameters/calibrate/preloop.gms:661]: Still useless brackets.

File: modules/29_CES_parameters/calibrate/preloop.gms (hunk 11 above)
[the remark this answers is not in the document]
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply modules/29_CES_parameters/calibrate/preloop.gms]: > * Use indentations to make the structure more readable

[[↑]](https://github.com/remindmodel/remind/blob/5c197bd42b7f4fc66f1bc9073896f632ea656f8a/main.gms#L144)

`cesOut2cesIn` is part of the summation index, hence indented with `t` inside the indexing bracket.
The rest is the summation term, hence indented two spaces from `sum`.

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]

**Pull request merged**